documentation.dd

```diff
@@ -0,0 +1,6 @@
+Ddoc
+
+$(D_S Documentation, $(NAVIGATION_DOCUMENTATION))
+
+Macros:
+    TITLE=Documentation
```

Comment:
That's the page if you open the documentation link: http://d-ag0aep6g.rhcloud.com/documentation.html
right-click "documentation" in the menu and open in a new tab to get there from the home page.